client/me/form-base/with-form-base.jsx

```diff
@@ -106,15 +64,28 @@ const withFormBase = ( WrappedComponent ) => {
 			toggleSetting: this.toggleSetting,
 			updateSetting: this.updateSetting,
 			submitForm: this.submitForm,
-			formState: this.state,
+			hasUnsavedUserSettings: this.props.hasUnsavedUserSettings,
 		} );
 
 		render() {
-			return <WrappedComponent { ...this.props } { ...this.getFormBaseProps() } />;
+			return (
+				<>
+					<QueryUserSettings />
+					<WrappedComponent { ...this.props } { ...this.getFormBaseProps() } />
+				</>
+			);
 		}
 	}
 
-	return connect( null, { errorNotice, successNotice } )( localize( EnhancedComponent ) );
+	return connect(
+		( state ) => ( {
+			_userSettings: getUserSettings( state ),
```

Comment:
Nit: I'm assuming you were using a `_` version of these props to indicate they're a reduxified version, but I'd suggest removing the leading underscores as it's generally against our coding guidelines and in the long run there's no good reason for it to be there.

Reply:
fixed via 01e03bc